### PR TITLE
Updated ModSDK scripts from various old versions to release-20200503

### PR DESCRIPTION
**Pull request by penev92**
This is an attempt to supersede #764 and update the mod to the current release (`release-20210321`). This will be followed by ~~3 other PRs~~ another PR that will update ~~things in pieces~~ everything (SDK scripts, engine/custom code and mod rules) from `release-20200503` to `release-20210321`. See commit messages for detailed explanations.
We are starting off small because some of the SDK scripts were quite outdated and this seems like low-hanging fruit.

### CODE REVIEW

File: packaging/windows/buildpackage.nsi [hunk 1]
```diff
@@ -14,6 +14,7 @@
 ;  You should have received a copy of the GNU General Public License
 ;  along with OpenRA.  If not, see <http://www.gnu.org/licenses/>.
 
+
```
Comment (abcdefg30) [inline packaging/windows/buildpackage.nsi:17]:
```suggestion
```

**Comment by pchote** [comment]
These files are no longer used and can also be deleted:
> packaging/linux/include/Eluant.dll.config
> packaging/linux/include/OpenAL-CS.dll.config
> packaging/linux/include/SDL2-CS.dll.config

File: launch-dedicated.sh [hunk 2]
```diff
@@ -54,5 +64,13 @@ while true; do
...
      Server.Name="${NAME}" Server.ListenPort="${LISTEN_PORT}" \
      Server.AdvertiseOnline="${ADVERTISE_ONLINE}" \
-     Server.EnableSingleplayer="${ENABLE_SINGLE_PLAYER}" Server.Password="${PASSWORD}"
+     Server.Password="${PASSWORD}" \
+     Server.RequireAuthentication="${REQUIRE_AUTHENTICATION}" \
+     Server.ProfileIDBlacklist="${PROFILE_ID_BLACKLIST}" \
+     Server.ProfileIDWhitelist="${PROFILE_ID_WHITELIST}" \
+     Server.EnableSingleplayer="${ENABLE_SINGLE_PLAYER}" \
+     Server.EnableSyncReports="${ENABLE_SYNC_REPORTS}" \
+	 Server.EnableGeoIP="${ENABLE_GEOIP}" \
```
Comment (pchote) [inline launch-dedicated.sh:73]: Whitespace error

**Review by pchote — COMMENTED** [review]
The SDK has the following files commited with unix line endings, it would be good IMO to change them here to match:
> launch-game.cmd
> make.ps1
> packaging/windows/buildpackage.nsi
> utility.cmd

packaging/linux/buildpackage.sh is also missing its end-of-file newline.

**Comment by pchote** [comment]
LGTM otherwise; there are some wording changes and a couple of other differences where the SDK is wrong, but if it isn't a goal to directly use the SDK files then that isn't an issue.

**Comment by penev92** [comment]
Removed old files
> packaging/linux/include/Eluant.dll.config
packaging/linux/include/OpenAL-CS.dll.config
packaging/linux/include/SDL2-CS.dll.config

and fixed indentation error in `launch-dedicated.sh`.

If you see any other discrepencies between this and https://github.com/OpenRA/OpenRAModSDK/tree/20200503 please be specific, as there are some intentional ones where the RA2 scripts have modified messages. There are also a few points where the ModSDK scripts may need fixing upstream, but since this is an old version of the ModSDK I've just skipped past that.

Also if someone not on Windows can handle the EoL thing (preferably at a later point when we're done because there are also issues inside the custom code) that would be great.

**Comment by penev92** [comment]
Added @pchote's commit to fix files for unix standards and added some missing artwork (used placeholders from the ModSDK example).

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**